libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -43,7 +43,12 @@
 #define AP_ARMING_COMPASS_MAGFIELD_MAX  875     // 1.65 * 530 milligauss
 #define AP_ARMING_BOARD_VOLTAGE_MAX     5.8f
 #define AP_ARMING_ACCEL_ERROR_THRESHOLD 0.75f
-#define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS
+#define AP_ARMING_AHRS_GPS_ERROR_MAX    10      // accept up to 10m difference between AHRS and GPS horizontally
+
+#define AP_ARMING_AHRS_GPS_ERROR_Z_MAX_CM   5000      // accept up to 50m difference between AHRS and GPS vertically
```

Comment:
Nitpick: Can we move these to `const uint32_t`'s inside the GPS check function? The separation of trying to go find where we defined the magic value at the top is a bit annoying IMO. (This is admittedly a bit of a style thing)

Reply:
Will move them, but you get to do argue the stylistic benefits with the other devs :-P